src/components/OasisMarketWidget.js

```diff
@@ -1,16 +1,82 @@
-import React, { PureComponent } from 'react'
+import React, { PureComponent } from 'react';
 import { PropTypes } from 'prop-types';
+import { last } from 'lodash';
+
 import OasisWidgetFrame from '../containers/OasisWidgetFrame';
+import { OasisTable } from './OasisTable';
+import { ETH_UNIT_ETHER } from '../constants';
+import { TIME_SPAN_DAY, TIME_SPAN_MONTH, TIME_SPAN_WEEK } from '../utils/timeSpan';
```

Comment:
do we need to have it like TIME_SPAN_DAY ... can't we have it just like DAY / MONTH / WEEK. Those are port of the time module so they are pretty isolated. If there are collisions  with other modules ( which I doubt ) we can use `as`

Reply:
done